extensions/3DTILES_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1623 @@
+- [Concepts](#concepts)
+  - [Schemas](#schemas)
+  - [Tileset Metadata](#tileset-metadata)
+  - [Tile Metadata](#tile-metadata)
+    - [Implicit Tile Metadata](#implicit-tile-metadata)
+  - [Group Metadata](#group-metadata)
+  - [Feature Metadata](#feature-metadata)
+  - [Statistics](#statistics)
+  - [Semantics](#semantics)
+  - [Styling](#styling)
+- [JSON Schema Reference](#json-schema-reference)
+
+## Overview
+
+This extension defines a standard mechanism for adding metadata to 3D Tiles. This includes:
+
+* Tileset metadata - metadata about the tileset as a whole.
+* Tile metadata - metadata about individual tiles.
+* Group metadata - metadata about groups of content.
+* Feature metadata - metadata about features. See the companion glTF extension [EXT_feature_metadata](https://github.com/CesiumGS/glTF/tree/master/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0).
+
+<img src="figures/metadata-granularity.png"  alt="Metadata Granularity" width="600">
+
+A tileset defines a **schema**. A schema has a set of **classes** and **enums**. A class contains a set of **properties**, which may be numeric, boolean, string, enum, or array types.
+
+**Entities** (such as tiles, features, etc.) conform to classes and contain **property values**. Depending on the context, property values may be stored in JSON or binary.
+
+**Statistics** provide aggregate information about the metadata. For example, statistics may include the min/max values of a numeric property for mapping property values to color ramps with the [declarative styling language](../../../specification/Styling) or the number of enum occurrences for creating histograms.
+
+By default, properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. The full list of built-in semantics can be found in the [Cesium Metadata Semantic Reference](../../../specification/Metadata/Semantics). Tileset authors may define their own application- or domain-specific semantics separately.
+
+This extension implements the [Cesium 3D Metadata Specification](../../../specification/Metadata/1.0.0), which describes the metadata format in full detail.
+
+## Use Cases
+
+_This section is non-normative_
+
+This extension is designed with several new use cases in mind.
+
+* An application can style tilesets using metadata stored at per-tileset, per-tile, per-group, and per-feature granularity
+* An application can see what metadata is available before requesting content, e.g. for populating a UI
+* Tileset authors can extend metadata with their own semantic data model
+* Metadata schemas can be shared within a tileset and across different tilesets
+* A runtime engine can optimize traversal algorithms by using per-tile metadata
+* An application can create a layering system by adding show/hide or color styles for groups, as well as selectively loading groups on demand
+
+## Compatibility Notes
```

Comment:
Seems like this should be combined with the **Dependencies** section.

> Instead use glTF content directly with [`3DTILES_content_gltf`](../../3DTILES_content_gltf/0.0.0).

The reader might think that b3dm and i3dm should use 3DTILES_content_gltf; instead, I think this means that you need to use 3DTILES_content_gltf to use this extension for content, is that right?

Please make things super explicit.